### PR TITLE
Script - wait_template 'trigger.entity_id' support

### PR DESCRIPTION
**Pull request by cdce8p**
**Description:**
Added `trigger.entity_id` and `variables` support for wait_template

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant):** home-assistant/home-assistant#9807

### CODE REVIEW

**Comment by frenck** [comment]
@cdce8p Could you please rebase this one onto the `next` branch?

File: source/_docs/scripts.markdown [hunk 1]
```diff
@@ -98,7 +98,7 @@ wait_template: "{{ is_state(trigger.entity_id, 'on') }}"
 ```
 {% endraw %}
 
-It is also possible to use dummy variables, e.g. in scripts, when using `wait_template`.
+It is also possible to use dummy variables, e.g., in scripts, when using `wait_template`.
```
Comment (cdce8p) [inline source/_docs/scripts.markdown]: Check out [Link](https://jakubmarian.com/comma-after-i-e-and-e-g/). Both versions are possible. I just tend to like the British one better.

**Comment by cdce8p** [comment]
@frenck I would suggest to leave it at `current` and just merge it after the next version of HA is public. Since it is only an extension to the current functions and not a new platform, I figured that `next`would not be right here.

File: source/_docs/scripts.markdown (hunk 1 above)
Comment (frenck) [reply source/_docs/scripts.markdown]: It OK to like the British one better, but we have a policy on having American English only.

**Pull request merged**